src/cmd/build.js

```diff
@@ -104,7 +104,7 @@ async function defaultPackageCreator(
   let packageName = safeFileName(
     `${extensionName}-${manifestData.version}.zip`);
   let extensionPath = path.join(artifactsDir, packageName);
-  let stream = createWriteStream(extensionPath);
+  let stream = createWriteStream(extensionPath, {flags:'wx'});
```

Comment:
I know this is a work in progress but I think changing the write flags isn't the best approach because it will show a confusing error to the developer. I'd rather see a check using [fs.stat()](https://nodejs.org/api/fs.html#fs_fs_stat_path_callback) first so that we can provide an informative error message like 'the build file destination ${extensionPath} already exists'